### PR TITLE
TMIB 32

### PR DESCRIPTION
**Pull request by notriddle**
Want to help out with writing TMIB but don't have write access to the repo?
[Go here](https://github.com/bors-ng/bors-ng.github.io/blob/tmib-32/_posts/2019-09-01-tmib-32.md), then click the pencil button. That'll open a pull request for the in-progress edition of TMIB.

These steps can be done immediately after opening the TMIB pull request:

- [x] Fix the "Go here" link at the top of this pull request

- [x] Update the issue number in the `title:` part

- [x] Update the month in the `date:` part

- [x] Update the month in the `summary:`

- [x] Update the URL in the "last month" PR link

These steps need to be done as the month rolls along, though they should probably be proofread at the end:

- [x] Write the "notable additions" list

- [x] Write the "new contributors" list

- [x] Find a new featured user

And, these should be done last thing:

- [x] Update the PR count in the PR paragraph

- [x] Create, and link in, next month's PR, so that the published edition links to the next one

- [x] Write a few useful tidbits in the summary: part

And publish to third-party syndication services:

- [MailChimp](https://us19.admin.mailchimp.com/lists/): @notriddle should copy and paste the newsletter, then tweak the links

### CODE REVIEW

**Comment by notriddle** [comment]
@alevy @ppannuto @niklasad1 Could you all review the "featured user" section?

File: _posts/2019-09-01-tmib-32.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+No new contributors this month 😢
+
+Want to see your name in this newsletter? Contribute on our [GitHub repository](https://github.com/bors-ng/bors-ng).
+
+
+### Who's using bors?
+
+This month's featured user is [Tock OS](https://www.tockos.org/), an embedded operating system for ARM Cortex-M microcontrollers. [Here's their GitHub page](https://github.com/tock/tock).
+
+Tock [is designed](https://www.tockos.org/documentation/design) to provide high-granularity isolation without using lots of memory. While most operating systems do isolation by giving each process its own slice of memory, Tock isolates "capsules" at the language level, allowing allocations to be mixed freely in memory and reducing fragmentation.
```
Comment (alevy) [inline _posts/2019-09-01-tmib-32.md]: This is a great descript, _except_ that Tock does both: "capsules" are the isolation unit in the kernel (so most akin to microkernel services in function) while applications are still isolated using more-or-less traditional processes.

Having said that, I won't be upset if it stays like this. Thanks for the shout out!

Comment (notriddle) [reply _posts/2019-09-01-tmib-32.md]: Possible improved description, then:

> Tock is designed to provide high-granularity isolation without using lots of memory. While most operating systems do isolation by giving each process its own slice of memory, Tock additionally isolates kernel-level "capsules" using language-based isolation, providing greater security guarantees without additional memory fragmentation.

Comment (alevy) [reply _posts/2019-09-01-tmib-32.md]: Perfect!

**Pull request merged**